### ISSUE TITLE
can model converage on 4 hour Cantonese dataset without punctuation?

### ISSUE THREAD

**Issue by Clement-Hui** [NVIDIA/tacotron2#256]
The symbols.py and cleaners.py have been changed minimally, and the model is now in around 20000 steps, still no signs of attention learning. The dataset contains no punctuation, and am training on batch size 4 on RTX 2060 super. Will attention ever converage? Thank you.
Please ignore the steps as I ahve resumed training before.
Training loss:
![Screenshot from 2019-08-24 22-44-08](https://user-images.githubusercontent.com/23010176/63638954-eafd4c80-c6c0-11e9-88e4-c12faac1c95b.png)
Attention and spectrograms:
![Screenshot from 2019-08-24 22-44-30](https://user-images.githubusercontent.com/23010176/63638960-f51f4b00-c6c0-11e9-88d2-41253029847c.png)
![Screenshot from 2019-08-24 22-44-38](https://user-images.githubusercontent.com/23010176/63638962-f781a500-c6c0-11e9-8eea-8b9598ada5c9.png)

**Comment by Clement-Hui** [comment]
I have trained and attention converge perfectly. Thank you for your attention

**Comment by delgerdalai** [comment]
@Clement-Hui how many steps did you train?

**Comment by Clement-Hui** [comment]
I trained for around 10k to 20k steps and attention converges around 50-70k. Around 10k and more model is overfitting the dataset. The result are not bad, audible at least

**Comment by terryyizhong** [comment]
> I have trained and attention converge perfectly. Thank you for your attention. I changed the batch size to 8 and run it on pretrained model to start training.

you said you run it on pretrained model. Does this pretrained model means the english model trained with LJspeech? How can you train a Cantonese model use a pretrained english model? @Clement-Hui

**Comment by Clement-Hui** [comment]
Please refer to my reply in issue #254
